src/sentry/services/hybrid_cloud/organization/impl.py

```diff
@@ -281,6 +282,18 @@ def _query_organizations(
 
         return [r.organization for r in results]
 
+    def update_flags(self, *, organization_id: int, flags: RpcOrganizationFlagsUpdate) -> None:
+        updates = models.F("flags")
+        for (name, value) in flags.items():
+            if value is True:
+                updates = updates.bitor(Organization.flags[name])
+            elif value is False:
+                updates = updates.bitand(~Organization.flags[name])
+            else:
+                raise TypeError
```

Comment:
```suggestion
                raise TypeError(f"Invalid value {value} received for update_flags")
```

Might be helpful for future us.